manager/cve_handler.py

```diff
@@ -151,14 +151,14 @@ def _id_query(rh_account_id, synopsis):
                         InsightsRule.name.alias('rule_id'),
                         SystemVulnerabilities.status_id.alias('status_id'),
                         SystemVulnerabilities.status_text.alias('status_text'),
+                        SystemVulnerabilities.mitigation_reason,
                         )
                 .join(SystemPlatform, on=(SystemVulnerabilities.system_id == SystemPlatform.id))
                 .join(CveMetadata, on=(SystemVulnerabilities.cve_id == CveMetadata.id))
                 .join(InsightsRule, JOIN.LEFT_OUTER, on=(InsightsRule.id == SystemVulnerabilities.rule_id))
                 .where(CveMetadata.cve == synopsis)
                 .where(SystemVulnerabilities.rh_account_id == rh_account_id)
-                .where((SystemVulnerabilities.mitigation_reason.is_null(True)) | (InsightsRule.active == False) & (~InsightsRule.rule_only))
-                .where((SystemVulnerabilities.when_mitigated.is_null(True)) | (InsightsRule.active == True) & (~InsightsRule.rule_only))
+                .where((SystemVulnerabilities.when_mitigated.is_null(True)) | (InsightsRule.active == True))
```

Comment:
This in the end should be.
```suggestion
                .where((SystemVulnerabilities.when_mitigated.is_null(True)) | ((InsightsRule.active == True) & (InsightsRule.rule_only == True)))
```
Please change